library/std/src/os/unix/io/mod.rs

```diff
@@ -23,22 +23,33 @@
 //! either by adding `unsafe` to APIs that dereference `RawFd` values, or by
 //! using to `BorrowedFd` or `OwnedFd` instead.
 //!
+//! The use of `Arc` for borrowed/owned file descriptors may be surprising.
+//! Unix file descriptors are mere references to internal *file descriptions*,
```

Comment:
Can't say I'm a huge fan of the term "file description" (it's too similar to "file descriptor"), but I guess we're stuck with it if that's what Posix uses.

Reply:
The full name is [open file description](https://pubs.opengroup.org/onlinepubs/9699919799/basedefs/V1_chap03.html#tag_03_258) which might be slightly less confusing? Maybe saying "kernel descriptor" or "kernel file descriptor" would help clarify. But sticking with the nomenclature used elsewhere does make sense.

Reply:
I have added the "open".